feat: allow configuration of ACME certificates duration (#8046)
traefik/traefik: docs/content/reference/static-configuration/env-ref.md

`TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_KEYTYPE`:  
KeyType used for generating certificate private key. Allow value 'EC256', 'EC384', 'RSA2048', 'RSA4096', 'RSA8192'. (Default: ```RSA4096```)

> `TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_RENEWBEFOREEXPIRY`:
> Number of hours left on a certificate before its expiry to renew it. (Default: ```720```)

Reviewer: ```suggestion
`TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_RENEWBEFOREEXPIRY`:
Time remaining before the certificate expiration when Traefik will try to renew it. (Default: ```720h0m0s```)
```

`TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_EXPIRATIONCHECKINTERVAL`:
Frequency in which Traefik will check if certificate is due for renewal. (Default: ```24h0m0s```)

> 

Author: This file has a header that says:

CODE GENERATED AUTOMATICALLY
THIS FILE MUST NOT BE EDITED BY HAND

So I'm removing this change.

Reviewer 2: This file must be generated and committed in your PR.

```console
$ go generate
```

> `TRAEFIK_CERTIFICATERESOLVERS_<NAME>_ACME_EXPIRATIONCHECKINTERVAL`:
> Frequency in which Traefik will check if issued certificates are due for renewal, expressed in hours. (Default: ```24```)
> 

Reviewer: ```suggestion
`TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_EXPIRATIONCHECKINTERVAL`:
Frequency in which Traefik will check if certificate is due for renewal. (Default: ```24h0m0s```)

```
